mrp_unbuild_cust_alu_analytics/models/mrp_unbuild_process_type.py

```diff
@@ -7,6 +7,11 @@ class MrpUnbuildProcessType(models.Model):
     _name = "mrp.unbuild.process.type"
     _description = 'Unbuild Process Type'
 
+    company_id = fields.Many2one(
+        comodel_name="res.company",
+        default=lambda self: self.env.company,
```

Comment:
add index for the field, stuff required for a good company access rule performance
```suggestion
        default=lambda self: self.env.company,
        index=True,
```